### PR TITLE
Fix #18

### LINKED ISSUE #18: NPE when iterating through rows
Trying to upgrade to 0.2.10 from 0.2.8, now that #15 was fixed...
java.lang.NullPointerException
	at com.monitorjbl.xlsx.StreamingReader.unformattedContents(StreamingReader.java:216)
	at com.monitorjbl.xlsx.StreamingReader.handleEvent(StreamingReader.java:136)
	at com.monitorjbl.xlsx.StreamingReader.getRow(StreamingReader.java:89)
	at com.monitorjbl.xlsx.StreamingReader.access$400(StreamingReader.java:55)
	at com.monitorjbl.xlsx.StreamingReader$StreamingIterator.hasNext(StreamingReader.java:440)
	at com.monitorjbl.xlsx.StreamingReader$StreamingIterator.<init>(StreamingReader.java:434)
	at com.monitorjbl.xlsx.StreamingReader.iterator(StreamingReader.java:236)

### PR DESCRIPTION
**Pull request by lilyliuce**
Let the method getStringCellValue() return "" instead of null when a new cell created.

### CODE REVIEW

File: src/test/java/com/monitorjbl/xlsx/StreamingReaderTest.java [hunk 1]
```diff
@@ -417,4 +417,18 @@ public void testEncryption() throws Exception {
       }
     }
   }
+  
+  @Test
+  public void testStringCellValue () throws Exception {
```
Comment (monitorjbl) [inline src/test/java/com/monitorjbl/xlsx/StreamingReaderTest.java]: Can you use 2 space indents here to match the rest of the code?

Comment (lilyliuce) [inline src/test/java/com/monitorjbl/xlsx/StreamingReaderTest.java]: Which file?

Comment (monitorjbl) [inline src/test/java/com/monitorjbl/xlsx/StreamingReaderTest.java]: The one this comment is on: src/test/java/com/monitorjbl/xlsx/StreamingReaderTest.java

Comment (lilyliuce) [inline src/test/java/com/monitorjbl/xlsx/StreamingReaderTest.java]: Wait a sec, I will fix that in a minute.

Comment (lilyliuce) [inline src/test/java/com/monitorjbl/xlsx/StreamingReaderTest.java]: I commit the indent change.

**Comment by monitorjbl** [comment]
Still not fixed, but I'll just format it after it's merged. Thanks for the PR!

**Pull request merged**